src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,96 @@
+import cn from 'classnames';
+import { PageGhangeCallback } from '../../types/PageChangeCallback';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange(page: number | PageGhangeCallback): void;
```

Comment:
The same typo 'PageGhangeCallback' appears in the type of the 'onPageChange' parameter. It should be corrected to 'PageChangeCallback' to match the type definition.